### DISCUSSION TITLE
Syntax checker for custom styling

### DISCUSSION THREAD

**Discussion by scottw12** [vanjarosoftware/Vanjaro.Platform#1250] [Q&A] [answered]
I have noticed that several newer CSS properties are  highlighted as an error.
Is this expected and what CSS standards are applicable to Vanjaro?

![image](https://user-images.githubusercontent.com/6786299/118166414-10793f00-b3eb-11eb-84ac-879300e0a1aa.png)

**Comment by scottw12** [comment]
???? bumping

**Comment by Gaganbird** [comment, marked as answer]
I have confirmed the issue and fixed it in the https://github.com/vanjarosoftware/Vanjaro.Platform/pull/1363
